helpers/browser/driver.js

```diff
@@ -85,12 +85,43 @@ class ChromeProtocol {
     });
   }
 
-  evaluateScript(chrome) { /* TODO(paulirish): scriptStr parameter */
-    return new Promise(function(resolve, reject) {
-      chrome.Runtime.evaluate({
-        expression: 'alert(navigator.userAgent)',
-        contextId: 0
-      }, resolve);
+  evaluateScript(scriptStr) {
```

Comment:
rename `scriptStr`? Though it seems like it would be useful to accept strings here, for e.g. scripts that need to include more than one function. Not too bad to require caller to create the `wrappedScriptStr` instead